core/src/main/scala/org/apache/spark/internal/config/Kryo.scala

```diff
@@ -22,35 +22,43 @@ import org.apache.spark.network.util.ByteUnit
 private[spark] object Kryo {
 
   val KRYO_REGISTRATION_REQUIRED = ConfigBuilder("spark.kryo.registrationRequired")
+    .version("1.1.0")
     .booleanConf
     .createWithDefault(false)
 
   val KRYO_USER_REGISTRATORS = ConfigBuilder("spark.kryo.registrator")
+    .version("0.5.0")
```

Comment:
No JIRA ID, commit ID: 91c07a33d90ab0357e8713507134ecef5c14e28a#diff-792ed56b3398163fa14e8578549d0d98
This is not a release version, do we need to record it?